source/rst/geom_series.rst

```diff
@@ -619,20 +617,20 @@ below
     get_ipython().run_line_magic('matplotlib', 'inline')
     
     #true present value of a finite lease
-    def finite_lease_pv(T,g,r,x_0):
+    def finite_lease_pv(T, g, r, x_0):
       G=(1+g)
```

Comment:
Can we add spaces around the equals signs here? We just don't need spaces around = in function arguments.